src/test/scala/nl.knaw.dans.easy.validatebag/rules/metadata/MetadataRulesSpec.scala

```diff
@@ -180,6 +180,14 @@ class MetadataRulesSpec extends TestSupportFixture with CanConnectFixture {
       includedInErrorMsg = "Point with only one coordinate")
   }
 
+  "archisIdentifiersHaveAtMost10Characters" should "fail if archis identifiers have too long values" in {
+    testRuleViolation(
+      rule = archisIdentifiersHaveAtMost10Characters,
+      inputBag = "ddm-invalid-archis-identifiers",
+      includedInErrorMsg = "(1) Archis identifier is too long: niet kunnen vinden1\n(2) Archis identifier is too long: niet kunnen vinden2"
```

Comment:
Why not use Scala's neat support for multi-line strings?

Reply:
Looks much better indeed! See 28275b7.